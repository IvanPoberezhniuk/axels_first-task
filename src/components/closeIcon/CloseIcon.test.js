import { unmountComponentAtNode } from 'react-dom';

import { CloseIcon } from '../index';

const setUp = () => render(<CloseIcon />);
let component;

beforeEach(() => {
  component = setUp();
});

afterEach(() => {
  unmountComponentAtNode(component);
  component.remove();
  component = null;
});

describe('CloseIcon component', () => {
  it('SNAPSHOT', () => {
    expect(component).toMatchSnapshot();
  });
});

