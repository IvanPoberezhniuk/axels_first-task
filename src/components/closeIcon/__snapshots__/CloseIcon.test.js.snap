// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseIcon component SNAPSHOT 1`] = `
<div
  class="sc-dlfnbm fUxuLr"
/>
`;
